web/app/js/components/NamespaceLanding.jsx

```diff
@@ -144,12 +146,12 @@ class NamespaceLanding extends React.Component {
 
     let metrics = this.state.metricsByNs[namespace] || {};
     let noMetrics = _isEmpty(metrics.pod);
-
+    // metrics.pod.map(p => console.log(p.tcp));
```

Comment:
Did you mean to leave this noted out line in here?

Reply:
omg whooooops ty :)